chichiwang/chichiwang.dev
# Replace boilerplate icons with site icons

## Description
Linked to Issue: #2 
Replace the [boilerplate](https://github.com/chichiwang/tamsui) favicons with personal logo.

## Changes
* Replaced all image assets in `static/images/favicon` with personal logo versions

## Steps to QA
* Verify all image assets in `static/images/favicon` have been replaced
* Pull down and switch to this branch
* Run the app and verify in browser that the favicon has been swapped out in the browser tab
